org.compiere.db.oracle.provider/src/org/compiere/db/DB_Oracle.java

```diff
@@ -536,7 +536,30 @@ public String TO_NUMBER (BigDecimal number, int displayType)
         }
         return result.toString();
     }   //  TO_NUMBER
+    
+	/**
+	 *	@return string with right casting for JSON inserts
+	 */
+	public String getJSONCast () {
+		return "parse_json(?)";

```

Comment:
not 100% sure but it seems to me parse_json is for Oracle NoSQL DB ...

Reply:
should be "json(?)" instead.

Reply:
changed in a new commit